syntax = "proto3";
package role;





/*
PutPermissionRole请求
*/
message PutPermissionRoleRequest {
    /*
    角色名
    */
    string role = 1 ;
    /*
    角色id
    */
    string id = 2 ;
}

/*
PutPermissionRole返回
*/
message PutPermissionRoleResponse {
    /*
    修改用户数量
    */
    int32 count = 1 ;
}

/*
PutPermissionRoleApi返回
*/
message PutPermissionRoleResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    role.PutPermissionRoleResponse data = 4 ;
}

